package.json

```diff
@@ -40,9 +40,9 @@
     "prettier": "^2.8.8",
     "string-replace-loader": "^2.1.1",
     "ts-jest": "^23.10.5",
-    "ts-loader": "^5.3.1",
+    "ts-loader": "^8.4.0",
```

Comment:
The previous version cannot load `./@types/conviva/*.d.ts` files, but it works fine in 8.x.